supervisor/utils/blockbuster.py

```diff
@@ -0,0 +1,35 @@
+"""Activate and deactivate blockbuster for finding blocking I/O."""
+
+from functools import lru_cache
+import logging
+
+from blockbuster import BlockBuster
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+@lru_cache(maxsize=1)
```

Comment:
```suggestion
@cache
```

Reply:
It looks like you intend to cache this forever?

Reply:
yea I did, will change, thanks